event-processing/lambda/tests/unit/app.event-processor.test.ts

```diff
@@ -584,7 +584,7 @@ describe('Unit test for app eventProcessorHandler', function () {
         expect(consoleMock).toHaveBeenCalledTimes(3);
         expect(consoleMock).toHaveBeenNthCalledWith(
             1,
-            '[WARN] UNKNOWN FIELDS\n{"sqsResourceName":"arn:aws:sqs:us-west-2:123456789012:SQSQueue","eventId":"66258f3e-82fc-4f61-9ba0-62424e1f06b4","eventName":"AUTHENTICATION_ATTEMPT","timestamp":"1609462861","message":"Unknown fields in message.","unknownFields":[{"key":"new_unknown_field","fieldName":"AuditEvent"},{"key":"unknown_user_field","fieldName":"User"}]}',
+            '[WARN] UNKNOWN FIELDS\n{"resourceName":"arn:aws:sqs:us-west-2:123456789012:SQSQueue","eventId":"66258f3e-82fc-4f61-9ba0-62424e1f06b4","eventName":"AUTHENTICATION_ATTEMPT","timestamp":"1609462861","message":"Unknown fields in message.","unknownFields":[{"key":"new_unknown_field","fieldName":"AuditEvent"},{"key":"unknown_user_field","fieldName":"User"}]}',
```

Comment:
looks like you are logging JSON, perhaps a dedicated JSON logger might be useful: https://github.com/pinojs/pino. not required, maybe add as a tech debt ticket